libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -400,7 +398,12 @@ bool AP_Arming::battery_checks(bool report)
             return false;
         }
 
-        for (int i = 0; i < _battery.num_instances(); i++) {
+        // all following checks are bypassed if USB is connected
+        if (hal.gpio->usb_connected()) {
```

Comment:
@WickedShell, I checked with Tridge and it turns out there isn't a parameter.  Instead in plane (and perhaps Rover) the battery failsafe can be triggered while USB is connected.  I think for now I will move this usb check up into the Copter's AP_Arming class.